src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/CamundaVariableDefinition.java

```diff
@@ -18,7 +18,9 @@
     TASK_STATE("taskState"),
     TASK_SYSTEM("taskSystem"),
     TITLE("title"),
-    TYPE("type");
+    TYPE("type"),
+    HAS_WARNINGS("hasWarnings");
```

Comment:
Can we update the Functional test/tests to show that when a task is returned it has the hasWarnings attribute?